MIGRATION_GUIDE.md

```diff
@@ -9,10 +9,54 @@ across different versions.
 
 ## v0.97.0 ➞ v0.98.0
 
-### *(new feature)* connection resource
-Added a new resource for managing connections.
+### *(new feature)* connection resources
+
+Added a new resources for managing connections. We decided to split connection into two separate resources based on whether the connection is primary or a replica (secondary). i.e.:
+
+- `snowflake_connection` is used as primary connection, with ability to enable failover to other accounts.
+- `snowflake_secondary_connection` is used as replica (secondary) connection.
+
+In order to promote secondary_connection to primary, resources need to be migrated (check [resource migration](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/docs/technical-documentation/resource_migration.md)) or re-created and imported using the following SQL statements on Snowflake Worksheet:
```

Comment:
This is good, but slightly rephrase that `In order to promote secondary_connection to primary, resources need to be removed from the state, re-created manually using the following SQL statements ... then imported again, but now as snowflake_connection (check resource migration).`